Marlin/src/lcd/e3v2/proui/dwin.cpp

```diff
@@ -1937,6 +1938,7 @@ void DWIN_RebootScreen() {
   DWINUI::Draw_CenteredString(Color_White, 220, GET_TEXT_F(MSG_PLEASE_WAIT_REBOOT));
   DWIN_UpdateLCD();
   delay(500);
+  DWIN_JPG_ShowAndCache(3)
```

Comment:
DWIN_JPG_ShowAndCache(3);

Reply:
Spotted last night but said screw it ill fix it in the AM and went to sleep! lol pushed now